packages/tests/src/web-test-runner.config.ts

```diff
@@ -138,7 +138,17 @@ See https://wiki.saucelabs.com/display/DOCS/Platform+Configurator for all option
     ...(browser === 'chromium'
       ? {
           launchOptions: {
-            args: ['--js-flags=--expose-gc', '--enable-precise-memory-info'],
+            args: [
+              '--js-flags=--expose-gc',
+              '--enable-precise-memory-info',
+              '--disable-background-timer-throttling',
+              '--disable-background-media-suspend',
+              '--disable-backgrounding-occluded-windows',
+              '--disable-input-event-activation-protection',
+              '--disable-renderer-backgrounding',
+              '--enable-automation',
+              '--disable-ipc-flooding-protection',
```

Comment:
where did these new flags come from?

Reply:
I found these in a few places that were documenting trying to work around Chrome's background tab throttling. I'm trying to serperate and/or revert the changes causing the problems to unblock this PR.